Dockerfile

```diff
@@ -1,23 +1,22 @@
-FROM python:3.5
+FROM python:3.6.2-slim
 
-USER root
+RUN useradd -ms /bin/bash serenata_de_amor
+WORKDIR /home/serenata_de_amor/rosie
 
 RUN apt-get update && apt-get install -y \
   build-essential \
   libxml2-dev \
   libxslt1-dev \
   python3-dev \
   unzip \
-  zlib1g-dev
-  
+  zlib1g-dev \
+  && rm -rf /var/lib/apt/lists/*
+
 RUN pip install --upgrade pip
 
+COPY config.ini.example ./
 COPY requirements.txt ./
 COPY setup ./
-COPY rosie.py ./
-COPY rosie ./rosie
-COPY config.ini.example ./
-
 RUN ./setup
 
-CMD python rosie.py run
```

Comment:
Running `docker run --rm -v /tmp/serenata-data:/tmp/serenata-data rosie` as stated in the docs doesn't produce anything. I guess this `CMD` line might have been deleted by accident (or new instructions should be provided).